cmd/thanos/query.go

```diff
@@ -85,7 +85,9 @@ func registerQuery(m map[string]setupFunc, app *kingpin.Application, name string
 	dnsSDInterval := modelDuration(cmd.Flag("store.sd-dns-interval", "Interval between DNS resolutions.").
 		Default("30s"))
 
-	enableAutodownsampling := cmd.Flag("query.auto-downsampling", "Enable automatic adjustment (step / 5) to what source of data should be used in store gateways if no max_source_resolution param is specified. ").
+	unhealthyStoreTimeout := cmd.Flag("store.unhealthy-timeout", "Timeout in minutes before a store is cleaned from the store UI page.").Default("5m").Duration()
```

Comment:
Can we use `modelDuration` here like everywhere else and then update the help message? (:

Reply:
Woah, don't know how I missed that :fearful: 

What's wrong with the help message ?

Reply:
You aren't forced to only use minutes when `modelDuration` is used 😄

Reply:
Yes ! You're right :smile: